pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
Do you mean the current value of PC, i.e. `pwndbg.regs.pc`, or the literal string `"$pc"`?  The former works fine, the latter should never work.

```
>>> pwndbg.disasm.one(pwndbg.regs.pc)
<capstone.CsInsn object at 0x7f62f89f64e0>
```